README.md

```diff
@@ -93,3 +93,97 @@ extension BannerView: BannerRendering {
 }
 ```
 
+## 🛰 Interactor
+
+> **Interactor**
+> (*noun*)
+> 
+> Component answering the question: What should I do in reaction to user action. 
+
+### Definitions
+
+### TL;DR
+
+Interactor is a final class that ViewController or View owns, which performs actions in reaction to user input - gestures, text input, device movement, geographical location change etc.
+
+#### Interactor: 
+- active component (has a lifecycle), 
+- final class (1),
+- can use other components for data retrieval,
+- can convert Model(s) to Renderable(s) ex. via map,
+- should call render methods on weakly held views (seen via protocol).
+
+*(1) There is no good reason for inheritance of custom classes, ever.*
+
+#### ViewControllers and Views:
+
+- can have many Interactors,
+- should OWN interactor and see it via protocol,
+- subviews of VC could own separate Interactor but it’s not mandatory for simple VC.
+
+### Example
+
+**Step 1 - Create Interactor protocols:**
+
+```swift
+protocol PodBayDoorsInteracting: class {
+	func use(_ banner: BannerRendering)
```

Comment:
I guess you should use the same indent width for all example source codes. Right now you use 2, 4 and 8 spaces alternately.

Reply:
So true.
I used like 3 tools to write it including Markdown editors - will fix it soon.